### PR TITLE
chore: added global test data file

### PR DESCRIPTION
**Pull request by guoqqqi**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

- Related issues

___
### New feature or improvement
- Describe the details and related test reports.
Move all common data to a public file
___

### CODE REVIEW

File: web/cypress/fixtures/data.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "editServiceSuccess": "Edit Service Successfully",
+  "deleteServiceSuccess": "Delete Service Successfully",
+  "deleteRouteSuccess": "Delete Route Successfully",
+  "submitSuccess": "Submit Successfully",
+  "deleteUpstreamSuccess": "Delete Upstream Successfully",
+  "createConsumerSuccess": "Create Consumer Successfully",
+  "deleteConsumerSuccess": "Delete Consumer Successfully",
+  "upstreamName": "test_upstream",
+  "serviceName": "test_service",
+  "serviceName2": "test_service",
```
Comment (juzhiyuan) [inline web/cypress/fixtures/data.json:14]: the value is the same as `serviceName`?

File: web/cypress/fixtures/data.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  "routeName": "test_route",
+  "consumerrName": "test_consumer",
```
Comment (juzhiyuan) [inline web/cypress/fixtures/data.json:16]:
```suggestion
  "consumerName": "test_consumer",
```

File: web/cypress/fixtures/data.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  "ip1": "127.0.0.1",
+  "ip2": "127.0.0.2",
+  "host1": "11.11.11.11",
+  "host2": "12.12.12.12",
+  "host3": "10.10.10.10",
+  "description": "desc_by_autotes",
```
Comment (juzhiyuan) [inline web/cypress/fixtures/data.json:22]:
```suggestion
  "description": "desc_by_autotest",
```

File: web/cypress/fixtures/data.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
+{
+  "deletePluginSuccess": "Delete Plugin Successfully",
```
Comment (LiteSun) [inline web/cypress/fixtures/data.json:2]: Are these data reused? @guoqi

File: web/cypress/fixtures/data.json [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  "serviceName2": "test_service",
+  "routeName": "test_route",
+  "consumerrName": "test_consumer",
+  "ip1": "127.0.0.1",
+  "ip2": "127.0.0.2",
+  "host1": "11.11.11.11",
+  "host2": "12.12.12.12",
+  "host3": "10.10.10.10",
+  "description": "desc_by_autotes",
+  "description2": "new desc",
```
Comment (juzhiyuan) [inline web/cypress/fixtures/data.json:23]:
```suggestion
  "description2": "description2",
```

File: web/cypress/fixtures/data.json [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+  "grafanaAddress": "Grafana Address",
+  "grafanaExplanation1": "Grafana address should begin with HTTP or HTTPS",
+  "grafanaExplanation2": "Address is illegality",
```
Comment (juzhiyuan) [inline web/cypress/fixtures/data.json:26]:
```suggestion
  "grafanaExplanation2": "Address is invalid",
```

File: web/cypress/fixtures/data.json [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+  "updateSuccessfully": "Update Configuration Successfully",
+  "sslRemoveSuccessfully": "Remove target SSL successfully",
```
Comment (juzhiyuan) [inline web/cypress/fixtures/data.json:28]: `deleteSSLSuccess` keep the same as others.

**Review by juzhiyuan — APPROVED** [review]

File: web/cypress/fixtures/data.json (hunk 4 above)
Comment (LiteSun) [reply web/cypress/fixtures/data.json:2]: ping @guoqqqi

Comment (guoqqqi) [reply web/cypress/fixtures/data.json:2]: Yes, these data are reused

File: web/cypress/integration/route/create-route-with-upstream.spec.js [hunk 8]
```diff
@@ -90,15 +87,18 @@ context('Create Route with Upstream', () => {
...
     cy.contains('Goto List').click();
     cy.url().should('contains', 'routes/list');
 
     // check if the changes have been saved
     cy.get(this.domSelector.nameSelector).type(this.data.routeName);
     cy.contains('Search').click();
+
     cy.contains(this.data.routeName).siblings().contains('Edit').click();
-    cy.contains('Next').click();
+    // ensure it has already change to edit page
```
Comment (juzhiyuan) [inline web/cypress/integration/route/create-route-with-upstream.spec.js:99]: https://www.grammarly.com/

**Review by liuxiran — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**